doc/whats-new.rst

```diff
@@ -55,6 +55,8 @@ Breaking changes
 Deprecations
 ~~~~~~~~~~~~
 
+- Deprecate rasterio backend (:issue:`4697`, :pull:`5808`).
```

Comment:
If I remember correctly, it was not a backend but just the `xr.open_rasterio` function. If that's correct, I'd change this to:
```suggestion
- Deprecate :py:func:`open_rasterio` (:issue:`4697`, :pull:`5808`).
```